x11-wm/selectwm/selectwm-0.4.1.ebuild

```diff
@@ -1,27 +1,33 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit autotools eutils
+EAPI=6
+inherit autotools
 
 DESCRIPTION="window manager selector tool"
-HOMEPAGE="http://ordiluc.net/selectwm"
-SRC_URI="http://ordiluc.net/selectwm/${P}.tar.bz2"
+HOMEPAGE="https://ordiluc.net/selectwm"
+SRC_URI="https://ordiluc.net/selectwm/${P}.tar.bz2"
 
 LICENSE="GPL-2"
```

Comment:
License is gpl-2+

Reply:
thanks for pointing that out, fixed it :)